src/main/java/io/leonis/algieba/geometry/Rotation.java

```diff
@@ -0,0 +1,28 @@
+package io.leonis.algieba.geometry;
+
+import lombok.experimental.UtilityClass;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.factory.Nd4j;
+
+/**
+ * The Class Rotation.
+ *
+ * @author Rimon Oz
+ */
+@UtilityClass
+public class Rotation {
+
+  /**
+   * @param input    The input vector.
```

Comment:
Since you assume a particular shape of the vector i'd reccomend documenting it.

Reply:
`planarCartesian` implies a vector with two elements, but i'll update the javadoc for posterity